packaging/rhel/syslog-ng.spec

```
@@ -7,7 +7,7 @@ Summary: Next-generation syslog server
 
 Group: System Environment/Daemons
 License: GPLv2+
-URL: http://www.balabit.com/network-security/syslog-ng
+URL: http://www.syslog-ng.org
```

However syslog-ng.org has a redirect rule, we prefer to use syslog-ng.com
Otherwise I think this patch - is not essential but - can be merged.

```suggestion
URL: http://www.syslog-ng.com
```